observability/akka-http-metrics/src/test/suite/scala/com/daml/metrics/akkahttp/AkkaHttpMetricsSpec.scala

```diff
@@ -0,0 +1,621 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics.akkahttp
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import com.google.common.primitives.Bytes
+
+import akka.util.ByteString
+import akka.stream.scaladsl.Source
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.server.Directives
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.model.{
+  HttpRequest,
```

Comment:
These imports don't seem to be organized (i.e., they are not in alphabetical order).